packages/text-editor/package.json

```diff
@@ -35,6 +35,7 @@
 	},
 	"dependencies": {
 		"@giselle-sdk/data-type": "workspace:^",
+		"@giselle-sdk/workflow-utils": "workspace:^",
```

Comment:
"I don't want to create a dependency on workflow-utils just for defaultName. It might be better to create a package like node-utils and place node-factory and defaultName there.